src/components/Sidebar.tsx

```diff
@@ -22,7 +22,7 @@ const Sidebar: React.FC = () => {
 
   if (isLoading) return <p>Loading...</p>;
 
-  const projectPath = isUUUID(project.id) ? project.id : "local"
+  const projectPath = isUUUID(project.id) ? project.id : "LOCAL-project"
```

Comment:
Same caps/lowercase here :)
Maybe we can move constant somewhere and use it everywhere?